soh/src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c

```diff
@@ -35,7 +35,7 @@ void BossDodongo_DrawEffects(GlobalContext* globalCtx);
 void BossDodongo_UpdateEffects(GlobalContext* globalCtx);
 
 const ActorInit Boss_Dodongo_InitVars = {
-    ACTOR_EN_DODONGO,
+    ACTOR_BOSS_DODONGO,
```

Comment:
I see this enum was renamed from ACTOR_EN_DODONGO to ACTOR_BOSS_DODONGO, but Gohma's was renamed the opposite way? I'm sure this is probably authentic jank, but if you wouldn't mind elaborating I'd appreciate it.

Reply:
Actually I think I'm seeing it, King Dodongo had the regular Dodongo as it's actor ID and Gohma larvae had Queen Gohma's actor id. What effect would this have in game, if any? That seems like it would've caused some pretty big problems I would think.

Reply:
A look through the code tells me that these particular actors don't have their instance's IDs checked very often, so I guess this is harmless and a good idea to take care of here.

Reply:
Yep, it's a benign vanilla bug. I made sure to check that their IDs weren't used and to play through the areas where these actors were present to make sure bugs weren't introduced. I fixed the init vars so I could make the assumption that they were accurate while initializing the DB and that `actor->id` reflected the ActorDB entry for them. Without these changes, I'd have to do more record keeping.